### DISCUSSION TITLE
namedtuple methods

### DISCUSSION THREAD

**Discussion by billbreit** [micropython/micropython#15277] [Using MicroPython] [answered]
I'm trying to use namedtuple collections to the max in order to reduce memory.  Is there an equivalent of namedtuple._fields and namedtuple._asdict in MicroPython ?  The workarounds are fairly easy, but I noticed a compiler directive for asdict in the namedtuple_obj.c file, so it may be there and I can't find it.

Thanks.

Bill

**Comment by scruss** [comment, marked as answer]
`_asdict` requires **MICROPY_CONFIG_ROM_LEVEL_AT_LEAST_EVERYTHING** defined in `mpconfigport.h`. This doesn't seem to be enabled anywhere.
